source/plugins/ruby/in_kube_pvinventory.rb

```diff
@@ -145,13 +151,8 @@ def parse_and_emit_records(pvInventory, batchTime = Time.utc.iso8601)
         end
 
         records.each do |record|
-          if !record.nil?
-            wrapper = {
-              "DataType" => "KUBE_PV_INVENTORY_BLOB",
-              "IPName" => "ContainerInsights",
-              "DataItems" => [record.each { |k, v| record[k] = v }],
-            }
-            eventStream.add(emitTime, wrapper) if wrapper
+          if !record.nil?          
+            eventStream.add(Fluent::Engine.now, record) 
```

Comment:
please remove all these irrelevant  time changes. Previously we get time once and use it in the emit. Now we are getting time for every record we emit. Even though we dont use emit time this is completely not required as its unnecessary to get time for every single record.,

Reply:
This shouldn't impact anything butI can update to call Fluent::Engine.now once ad use the same pattern as emitTime.